addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -65,7 +65,7 @@ private _id = switch (_type) do {
     };
     case "turretweapon": {
         if (_applyRetroactively) then {
-            [GVAR(oldUnit), GVAR(oldUnit) currentWeaponTurret (GVAR(oldUnit) call CBA_fnc_turretPath), ""] call _function;
+            [GVAR(oldUnit), GVAR(oldTurretWeapon), ""] call _function;
```

Comment:
Why?

Reply:
we have the value available in a variable, simplest and fastest way to return it